### ISSUE TITLE
in project view, display project info panel

### ISSUE THREAD

**Issue by 0xpentacle** [pentacledotai/pentacle#6]
when a site visitor has navigated into a project record view

a) the project sunburst pie to take a maximum of 0 - 270 degrees counter-clockwise
b) leaving a 90 degree gap top left for project info panel

![image](https://user-images.githubusercontent.com/80323528/119754216-4262c900-bef4-11eb-8ebf-9f1360bd31ee.png)

**Comment by 21e8** [comment]
Exactly which data do we wanna display here? Can we define that?

**Comment by 0xpentacle** [comment]
Just a div will do for now 

where we can add project name plus some details as in image

Noid has created some content which we can put into a json file

**Comment by 21e8** [comment]
Yeah so I could kinda wrap my head around how to do it but we gotta change quite a bit of code and there's quiet some math involved. Hope I can do it by next week
